gapic/src/main/com/google/gapid/server/GapisProcess.java

```diff
@@ -113,6 +128,10 @@ protected Exception prepare(ProcessBuilder pb) {
       args.add(adb);
     }
 
+    if (gapisArgs.isSpecified()) {
+      args.add(gapisArgs.get());
```

Comment:
Looks like you got yourself into a bit of a pickle... however, since we've already implemented a "split a string into flags list" function in ```core/text```, I'd suggest passing the string to gapis and having gapis use that function to split it apart. Assuming, of course, you can then pass the result on to the flag parsing code...

Otherwise, I do have code somewhere, that can collect multiple instances of a flag into a list. This would still limit the flag's syntax to the ```--name=value``` form:
```./do run gapic --gapis-arg '--foo=bar' --gapis-arg '--baz=meow'```
Otherwise you'd have to use this really clumsy form:
```./do run gapic --gapis-arg --foo --gapis-arg bar --gapis-arg --baz --gapis-arg meow```
Let me know if you want me to add in the string-list flag type.

Another ugly alternative is to re-implement ```core/text```'s splitting function in Java. Unfortunately, guava's ```Splitter``` does not yet support "splitting with escapes".